server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/routes/TasksRoutesTest.java

```diff
@@ -193,6 +193,68 @@ void getAwaitShouldAwaitTaskCompletion() {
             .body("status", is("completed"));
     }
 
+    @Test
+    void getAwaitWithTimeoutShouldAwaitTaskCompletion() {
+        TaskId taskId = taskManager.submit(() -> {
+            try {
+                Thread.sleep(2000);
+            } catch (InterruptedException e) {
+                throw new RuntimeException(e);
+            }
+            return Task.Result.COMPLETED;
+        });
+
+        given()
+            .queryParam("timeout", "5")
+        .when()
+            .get("/" + taskId.getValue() + "/await")
+            .then()
+            .statusCode(HttpStatus.OK_200)
+            .body("status", is("completed"));
+    }
+
+    @Test
+    void getAwaitWithInvalidTimeoutShouldReturnAnError() {
+        TaskId taskId = taskManager.submit(() -> Task.Result.COMPLETED);
+
+        given()
+            .queryParam("timeout", "-5")
+            .when()
```

Comment:
Indent

Reply:
That went back somehow in one of your fixup?

Reply:
a github bug I think